tests/unit/job/test_query.py

```diff
@@ -1139,114 +1156,92 @@ def test_result_with_none_timeout(self):
             timeout=None,
         )
 
-    def test_result_with_done_jobs_query_response_and_page_size_invalidates_cache(self):
```

Comment:
>         """We don't call jobs.query with a page size, so if the user explicitly
>        requests a certain size, invalidate the cache.
>        """

This is no longer true. We do set the page size in the initial getQueryResults call.